app/views/components/_govuk_tabs.html.erb

```diff
@@ -0,0 +1,40 @@
+<%
+  id_prefix = local_assigns[:idPrefix]
+  attributes = {
+    class: class_names("govuk-tabs", local_assigns[:classes]),
+    data: { module: "govuk-tabs" }
+  }.merge!(local_assigns[:attributes] || {})
+%>
+<%= tag.div **attributes do %>
+  <h2 class="govuk-tabs__title">
+    <%= local_assigns[:title] || "Contents" %>
+  </h2>
+  <% if (items = local_assigns[:items] || []).any? %>
+    <ul class="govuk-tabs__list">
+      <% items.each_with_index do |item, i| %>
+        <% if item %>
```

Comment:
Do we need `if item` here?

Reply:
I kept these for compatability with the original template to avoid any surprises if someone is expecting the same behaviour: https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/components/tabs/template.njk#L12